### PR TITLE
Cosmetic tweaks to native enqueue spec

### PR DESCRIPTION
**Pull request by EwanC**
Pedantic things I noticed while reading spec.

### CODE REVIEW

**Comment by kbenzie** [comment]
You should add yourself to the list of contributors, I notice the diff isn't up to date with main so rebase before you do that.

**Review by kbenzie — APPROVED** [review]

File: scripts/core/EXP-NATIVE-ENQUEUE.rst [hunk 1]
```diff
@@ -32,13 +32,13 @@ within the native API through the function passed to
...
 not the native queue returned by ${x}QueueGetNativeHandle results in undefined
-behaviour.
+behavior.
 
-Any args that are needed by the func must be passed through a void* and unpacked
-within the func. If ${x}_mem_handle_t arguments are to be used within
-pfnNativeEnqueue, they must be accessed using ${x}MemGetNativeHandle.
-${x}_mem_handle_t arguments must be packed in the void* argument that will be
-used in pfnNativeEnqueue, as well as ${x}EnqueueNativeCommandExp's phMemList
+Any args that are needed by the func must be passed through a `void*` and unpacked
```
Comment (aarongreig) [inline scripts/core/EXP-NATIVE-ENQUEUE.rst:37]: I think single backticks is italics, double backticks for monospace

Comment (kbenzie) [reply scripts/core/EXP-NATIVE-ENQUEUE.rst:37]: putting `${x}` in inline monospace doesn't render well though

Comment (aarongreig) [reply scripts/core/EXP-NATIVE-ENQUEUE.rst:37]: it seems like it's working ok in this instance https://github.com/oneapi-src/unified-runtime/blob/main/scripts/core/EXP-MULTI-DEVICE-COMPILE.rst?plain=1#L55 https://oneapi-src.github.io/unified-runtime/core/EXP-MULTI-DEVICE-COMPILE.html#support

**Pull request merged**